code/modules/mob/living/carbon/xenomorph/castes/warrior/abilities_warrior.dm

```diff
@@ -112,9 +112,14 @@
 
 /datum/action/xeno_action/activable/lunge/proc/lunge_grab(mob/living/carbon/xenomorph/warrior/X, atom/A)
 	X.remove_filter("warrior_lunge")
-	if (!X.Adjacent(A))
+	if(!X.Adjacent(A))
 		return
 
+	if(isliving(A))
+		var/mob/living/L = A
+		if(L.stat == DEAD)
+			return
```

Comment:
i'm not sure lunge can act on dead marines already.

Pretty sure the check is on start_pulling

Reply:
I'll take the lunge checks out for now. I may have encountered a false positive while reproducing, can't do so reliably, perhaps the round was not started or something.

Thanks mole.